.github/workflows/release.yml

```diff
@@ -20,11 +24,38 @@ jobs:
       - name: Install release dependencies
         run: |
           python -m pip install --upgrade pip
-          pip install setuptools wheel twine
+          pip install poetry==1.1.7
 
       - name: Build and publish package
         env:
-          TWINE_USERNAME: ${{ secrets.PYPI_STACUTILS_USERNAME }}
-          TWINE_PASSWORD: ${{ secrets.PYPI_STACUTILS_PASSWORD }}
+          POETRY_HTTP_BASIC_PYPI_USERNAME: ${{ secrets.PYPI_STACUTILS_USERNAME }}
+          POETRY_HTTP_BASIC_PYPI_PASSWORD: ${{ secrets.PYPI_STACUTILS_PASSWORD }}
         run: |
           scripts/cipublish
+
+      - name: Tag Release
+        uses: "marvinpinto/action-automatic-releases@v1.2.1"
```

Comment:
Hadn't seen this before. Looks useful, though I don't mind creating releases manually. This puts every commit into the release notes, which I think would make more sense if we were using squash merging of PRs (which maybe we should start doing). Happy to keep using this and see how it goes.